.insomnia/Request/req_6376c50552f14284a40c130940fba638.yml

```diff
@@ -1,19 +1,32 @@
 _id: req_6376c50552f14284a40c130940fba638
 type: Request
 parentId: fld_80bc7ee1b14547dc860545b5af4878c5
-modified: 1679174399656
+modified: 1687182015546
 created: 1643841355063
 url: "{{ _.base_url }}/api/v1/reviews"
 name: Reviews
 description: >
-  This endpoint lists all active (not archived) reviews the logged in user has
-  write access to. In this sense, it lists the following:
+  This endpoint lists all active or archived (depends on the `archive_state`
+  param) reviews the logged in user has write access to. In this sense, it lists
+  the following:
 
   1. `owned_reviews`: owned reviews
 
   1. `collab_reviews`: collaboration reviews
 
 
+  Params:
+
+  - `archive_state` when:  
+    - `live` or `nil`, the endpoint will return the live reviews.
+  	- `archived`, the endpoint will return the archived reviews.
+
+  - `page`: Pagination param, when:
+  	- `page` value is greater than `0`, the endpoint returns the specified page with page's length is `5`
```

Comment:
Pending merge for current PR, will change to `10`.

Reply:
Fixed.